### ISSUE TITLE
[FEATURE] Documentation for Build

### ISSUE THREAD

**Issue by nikunjsanghai** [debOliveira/MoCapRasp#22]
**Is your feature request related to a problem? Please describe.**
No.  

**Describe the solution you'd like**
From the existing documentation, it is not entirely clear *how* to operationalize the entire motion capture system. From our understanding, there is central script `mocaprasp.py`, and from here would call the following functions in order:
1. Camera Extrinsics Calibration
2. Ground Plane Estimation
3. Standard Capture Routine
Is this ordering correct?

**Describe alternatives you've considered**
NA

**Additional context**
NA

**Comment by debOliveira** [comment]
Yes, the ordering is correct. The instructions for running the calibration and capture are in the [README](https://github.com/debOliveira/MoCapRasp/tree/main/client#%EF%B8%8F-usage):

https://github.com/debOliveira/MoCapRasp/blob/2f8ce0318fc9aa95b24db2dcfacb7ebc4ab2af2f/client/README.md?plain=1#L64-L69

@loolirer can give you more details if something remains unclear.

**Comment by debOliveira** [comment]
@nikunjsanghai is your issue solved?

**Comment by adthoms** [comment]
@debOliveira thank you for the support. Both @nikunjsanghai and I are attempting to recreate the results in your [conference paper](https://ieeexplore.ieee.org/abstract/document/10053039) and so far things are going smoothly. We are waiting for our IR LED lights to arrive this week, and we will let you know if any issues arise. If you are curious, we are putting together the following experimental rig to evaluate a SLAM algorithm I have developed. 
![image](https://github.com/user-attachments/assets/47cbf70a-ba88-43a5-839b-daaf4d58876e)
As shown, we are using three Module 2 NoIR cameras fixed to a 3D-printed plate, and we aim to track a drone ($F_R$) with respect to the camera cluster ($F_C$) to (ultimately) resolve frame $F_J$ in the world coordinate frame ($F_W$).

**Comment by debOliveira** [comment]
Hey @adthoms,

That is excellent work! Please keep me posted when the paper comes out; we may apply it. Contact us if issues arise or if you want to brainstorm some ideas.

@loolirer developed a [digital twin](https://github.com/loolirer/VirtualMoCap) for our arena to facilitate prototyping. Maybe it is useful for some of your visualizations.

**Comment by adthoms** [comment]
Hello @debOliveira. One major issue we are facing is in capturing the 6 DoF pose of a collection of reflectors that do not match the calibration 1D wand illustrated in Fig 2 of your [conference paper](https://ieeexplore.ieee.org/document/10053039). Is it possible to track and associate markers in a configuration similar to the laboratory experiment we will illustrated above?

**Comment by debOliveira** [comment]
Hey @adthoms 

Yes, it is possible to track it, and you are correct to say it needs a new assignment  scheme. 

I don't have the time to go deep into this right now, but you can follow up with the @ieee-ras-ufcg group in case you need assistance. They are taking care of the arena and adding new wand and robot marker configurations. 

Cheers

**Comment by debOliveira** [comment]
I'm closing this issue due to inactivity.
